test/tags/computeColumn.js

```diff
@@ -0,0 +1,77 @@
+const Liquid = require("../..");
+const chai = require("chai");
+const expect = chai.expect;
+chai.use(require("chai-as-promised"));
+
+describe("tags/computeColumn", function () {
+  it("should compute column values", function () {
+    const liquid = Liquid();
+
+    const ctx = {
+      dynamicTable: [
+        { col1: 2, col2: 3, col3: undefined },
+        { col1: 1, col2: 4, col3: undefined },
+      ],
+      someOtherVariable: 10,
+    };
+    const src = `{% computeColumn dynamicTable col3 %}
+                      {% assign dummy = 1 %}
+                      {% assign dummy2 = 2 %}
+                      {% assign col3 = self.col1 | plus: self.col2 | plus: someOtherVariable %}
+                      {% if dummy == 1 %}
+                        {% assign dummy2 = 0 %}
+                      {% endif %}
+                      {% assign $$answer = col3 | plus: dummy2 | minus: someOtherVariable | append: ' Per month' %}
+                {% endcomputeColumn %}`;
```

Comment:
Just a thought, if this is supposed to be computed in the context of a table, can't we figure out the table name in code?

Reply:
yes that will happen in angulaR FE side